README.md

```diff
@@ -1,15 +1,15 @@
 # virtual-controlplane
 
+![Gardener on Metal Logo](docs/assets/logo.png)
+
 This project contains a collection of helm charts to deploy a virtual/nodeless Kubernetes API server.
 
-The API server can be deployed using various storage backends
+## Start using or developing the virual-controlplane locally
+
+Please see our [documentation](https://onmetal.github.io/virtual-controlplane/) or the content in the `/docs` folder for more details.
 
-* Etcd
-* Postgress
-* CockroachDB
+## Feedback and Support
 
-## Usage
+Feedback and contributions are always welcome!
 
-```shell
-helm repo add vc https://onmetal.github.io/virtual-controlplane/
-```
+Please report bugs, suggestions or post questions by opening a [Github issue](https://github.com/onmetal/virtual-controlplane/issues).
```

Comment:
In order to report bugs, submit suggestions or post questions please open a [Github issue](https://github.com/onmetal/virtual-controlplane/issues).